docs/headlampFAQ.md

```diff
@@ -0,0 +1,132 @@
+
+### Q: Is there any cost involved in using Headlamp?
+
+**A:** Headlamp is completely free to use. As an open source tool under the Apache 2.0 License, it is available at no charge, and users are encouraged to modify and redistribute it in accordance with the license terms.
+
+### Q: Can I use Headlamp for commercial purposes?
+
+**A:** Yes, Headlamp is licensed for both personal and commercial use. The Apache 2.0 License allows for the use of Headlamp in any setting, from individual projects to enterprise-level operations.
+
+### Q: Where can I find the source code for Headlamp?
+
+**A:** The source code for Headlamp is publicly available on [GitHub](https://github.com/headlamp-k8s/headlamp). You are welcome to explore, fork, and contribute to the codebase.
+
+### Q: Who maintains Headlamp, and how is it funded?
+
+**A:** Headlamp is currently maintained by the original Kinvolk team, now part of Microsoft. The integration has provided the team with access to greater resources, enhancing their ability to develop, maintain, and expand Headlamp's capabilities.
+
+### Q: What features does Headlamp offer?
+
+**A:** Headlamp provides a range of features designed to enhance the Kubernetes experience. These include a clean and informative dashboard, the ability to inspect and edit Kubernetes resources, real-time cluster metrics, and plugin support for extensibility.
+
+### Q: How does Headlamp ensure the security of my Kubernetes cluster?
+
+**A:** Headlamp prioritizes security by using your local kubeconfig file to communicate directly with your cluster, ensuring that access control is governed by your Kubernetes configurations and credentials. It does not store your credentials on any server, minimizing the risk of unauthorized access.
+
+### Q: Is Headlamp customizable?
+
+**A:** Yes, Headlamp is built with customization in mind. It offers a plugin system that allows you to extend its functionality to better suit your specific use cases and workflow requirements.
+
+### Q: How often is Headlamp updated?
+
+**A:** The Headlamp project is actively maintained, and updates are released regularly. These updates can include new features, bug fixes, and security enhancements to ensure that users have the best possible experience with the tool.
+
+---
+
+# Installation and Setup
+
+### Q: Where can I download the latest version of Headlamp?
+
+**A:** To get the latest version of Headlamp, you can fork or clone the repository directly from the [Headlamp GitHub page](https://github.com/headlamp-k8s/headlamp). This allows you to customize the code according to your needs before building and installing it.
+
+### Q: Are there any prerequisites for installing Headlamp?
+
+**A:** Yes, you need to have a Kubernetes cluster running and `kubectl` installed and configured on your machine. Ensure that you have the necessary permissions to view and manage the cluster resources.

```

Comment:
This is another question that can be confusing. I'd remove it.
(and we don't need kubectl in the machine for Headlamp to run)